odoo/tools/translate.py

```diff
@@ -257,7 +257,7 @@ def process(node):
             result.tail = node.tail
             has_text = (
                 todo_has_text or nonspace(result.text) or nonspace(result.tail)
-                or any(name in TRANSLATED_ATTRS for name in result.attrib)
+                or any((key in TRANSLATED_ATTRS and val) for key, val in result.attrib.items())
```

Comment:
```suggestion
                or any((key in TRANSLATED_ATTRS and val) for (key, val) in list(result.attrib.items()))
```
See https://docs.python.org/3.6/library/stdtypes.html#dict-views

Reply:
@mreficent why is it a problem to use views? what is the benefit to use a list?

Reply:
It's not a problem. It's just that it's more common to use lists everywhere.

Reply:
if it was for a long for loop and avoid future side effect, I would agree with you. Here it only complexify this (already too long) one liner.

Reply:
ok